resources/js/blocks/molliePaymentMethod.js

```diff
@@ -72,7 +72,7 @@ const MollieComponent = (props) => {
             method: 'POST',
             data: {
                 action: 'mollie_checkout_blocks_surchage',
-                method: activePaymentMethod
+                payment_method: activePaymentMethod
```

Comment:
Unclear why it was done, could be great to put changes like in a separate commit, with commit message/description explaining that. Useful not just for reviewing, but also when we need to check the history, `git blame` etc.